dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/metric_config_panel_item.tsx

```diff
@@ -0,0 +1,233 @@
+    }
+  }, [
+    data.defaultAxes,
+    data.rawVizData?.[visualizations.vis.name]?.dataConfig,
+    visualizations.vis.name,
+  ]);
+
+  const updateList = (value: string, index: number, name: string, field: string) => {
+    const listItem = { ...configList[name][index], [field]: value };
+
+    const updatedList = {
+      ...configList,
+      [name]: [
+        ...configList[name].slice(0, index),
+        listItem,
+        ...configList[name].slice(index + 1, configList[name].length),
+      ],
+    };
+    setConfigList(updatedList);
+  };
+
+  const updateListWithAgg = (value: string[], index: number, name: string, field: string) => {
+    const updatedList = {
+      ...configList,
+      [name]: [
+        ...configList[name].slice(0, index),
+        {
+          ...configList[name][index],
+          [field]: [...value],
+        },
+        ...configList[name].slice(index + 1, configList[name].length),
+      ],
+    };
+    setConfigList(updatedList);
+  };
+
+  const handleServiceRemove = (index: number, name: string) => {
+    const arr = [...configList[name]];
+    arr.splice(index, 1);
+    const updatedList = { ...configList, [name]: arr };
+    setConfigList(updatedList);
+  };
+
+  const handleServiceAdd = (name: string) => {
+    const updatedList = { ...configList, [name]: [...configList[name], initialConfigEntry] };
+    setConfigList(updatedList);
+  };
+
+  const updateChart = () => {
+    dispatch(
+      renderExplorerVis({
+        tabId,
+        data: {
+          ...explorerVisualizations,
+          [visualizations.vis.name]: {
+            dataConfig: {
+              metrics: configList.metrics,
+            },
+          },
+        },
+      })
+    );
+  };
+
+  const getOptionsAvailable = (sectionName: string) => {
+    const selectedFields = {};
+    for (const key in configList) {
+      configList[key] && configList[key].forEach((field) => (selectedFields[field.label] = true));
+    }
+
+    return fieldOptionList
+      .filter((field) => !selectedFields[field.label])
+      .filter((field) => numericalTypes.includes(field.type));
+  };
+
+  const getCommonUI = (lists, sectionName: string) =>
+    lists &&
+    lists.map((singleField, index: number) => (
+      <React.Fragment key={index}>
+        <div className="services">
+          <div className="first-division">
+            <EuiPanel color="subdued" style={{ padding: '0px' }}>
+              <EuiFormRow
+                label="Aggregation"
+                labelAppend={
+                  lists.length !== 1 && (
+                    <EuiText size="xs">
+                      <EuiIcon
+                        type="cross"
+                        color="danger"
+                        onClick={() => handleServiceRemove(index, sectionName)}
+                      />
+                    </EuiText>
+                  )
+                }
+              >
+                <EuiComboBox
+                  aria-label="Accessible screen reader label"
```

Comment:
Please mention proper label. `Accessible screen reader label` is a generic text (probably provided by the component vendor).